editor/src/components/menubar/menubar.tsx

```diff
@@ -218,7 +218,7 @@ export const Menubar = betterReactMemo('Menubar', () => {
           </a>
         </Tile>
 
-        <Tooltip title={'Project Info'} placement={'right'}>
```

Comment:
do we need to lose the placement prop? I guess it wouldn't be hard for someone to use the placement prop in the css of our in-house Tooltip, and restoring these later on would be a one+ hour chore of cherry picking and merge conflicts

Reply:
The removal is in this one commit, so it would be more like 10 minutes of copy-paste times: https://github.com/concrete-utopia/utopia/pull/1541/commits/e83d0f857d6899c24d71d78dda28c4496ac08358

But it was removed because the new tooltip doesn't support it and even the type itself was coming from Tippy.

Reply:
it's easier to reverse cherry-pick it while it's fresh and the code did not drift – even if we are not using the information in this PR

Reply:
That's with the assumption the support will be restored, but if it's left in it's a lie.

Reply:
my assumption is that if we merge orange tooltips to master, someone will need to fix it ASAP

Reply:
I wasn't aware we even _have_ a new tooltip - thought you were still investigating whether replacing it with the dummy one I sent you actually makes a performance improvement, or still suffers from the same re-render problems.

Reply:
OK then, it appears it was purely for diagnostic purposes, so I'll revert the changes.